### PR TITLE
Update applicability messages for clarity in tests

### PR DESCRIPTION
**Pull request by zanieb**
These names are only ever displayed internally right now and we could be clearer in our test snapshots.

The diff is kind of scary because all of the tests fixtures are updated.

### CODE REVIEW

**Comment by zanieb** [comment]
This is disruptive so I've been putting it off but might be worth it.

File: crates/ruff_linter/src/message/diff.rs [hunk 1]
```diff
@@ -54,9 +54,9 @@ impl Display for Diff<'_> {
 
         let message = match self.fix.applicability() {
             // TODO(zanieb): Adjust this messaging once it's user-facing
-            Applicability::Safe => "Fix",
-            Applicability::Unsafe => "Suggested fix",
-            Applicability::Display => "Possible fix",
+            Applicability::Safe => "Safe fix",
+            Applicability::Unsafe => "Unsafe fix",
+            Applicability::Display => "Display only fix",
```
Comment (charliermarsh) [inline crates/ruff_linter/src/message/diff.rs:59]: Nit: "Display-only fix"? As-is it reads as imperative to me, like an action.

Comment (zanieb) [reply crates/ruff_linter/src/message/diff.rs:59]: That works. Should we consider improving the name of that applicability level?

Comment (charliermarsh) [reply crates/ruff_linter/src/message/diff.rs:59]: To `DisplayOnly`?

Comment (charliermarsh) [reply crates/ruff_linter/src/message/diff.rs:59]: I'm fine with that

**Review by charliermarsh — APPROVED** [review]

**Pull request merged**